doc/sphinx/how_to_install.rst

```diff
@@ -7,7 +7,7 @@ Requirements
 ------------
 
 Before you install cclib, you need to make sure that you have the following:
- * Python (at least version 3.0 is recommended, although 2.7 is still tested)
+ * Python (at least version 3.4 is recommended, although 2.7 is still tested)
```

Comment:
Is "recommended" the right word here? Will something break with version <3.4?

Reply:
It means that 3.0, 3.1, 3.2, and 3.3 are not supported, but 2.7 is, and we prefer >= 3.4 over 2.7.

How about:

> Before you install cclib, you need to make sure that you have the following:
> * Python (at least version 3.4, although 2.7 is still tested)
> * NumPy (at least version 1.12)

That's the new minimum version of NumPy.

Reply:
Well, I'm trying to understand if cclib will still work for uers with Python < 3.4. If so, then "make sure that you have" is not the right wording. IIUC we want >= 3.4 for our own convenience.

Reply:
If we limit `periodictable` to the `Nuclear` method, then only that part would require 3.4. If the built-in periodic table gets replaced with `periodictable`, then all of our code will require 3.4.

Reply:
OK, I guess I'd prefer to make this recommended, not required. I like that cclib will still work with older Python version, even if some parts won't and we don't test older version (although we could test and exclude some modules for those versions).

Since we do have a number of "soft" dependencies, it'd be good to standardize the way we try to import them if we don't already. There are some options here, as discussed https://stackoverflow.com/questions/14050281/how-to-check-if-a-python-module-exists-without-importing-it/25045228.

Reply:
I've made a separate issue for the imports.

Even if it's possible, I'd rather not include language that implies we can run on a version of Python that hasn't received a security update in 4 years.

Reply:
OK